pylot/map/hd_map.py

```diff
@@ -317,9 +317,13 @@ def _must_obey_american_traffic_light(self, ego_transform: Transform,
         else:
             return (False, None)
 
-    def get_lane(self, location: Location, waypoint_precision: float = 0.05):
+    def get_lane(self, location, lane_id=0, waypoint_precision=0.05):
```

Comment:
Is there a reason why you've removed the type annotations?

Reply:
Sorry, the codebase I was testing on was outdated, and didn't have any of the type annotations. I'll make sure to add them back